spec/acceptance/suites/rpm/01_simp_server_spec.rb

```diff
@@ -0,0 +1,165 @@
+require_relative 'spec_helper_rpm'
+require 'erb'
+
+test_name 'puppetserver via rpm'
+
+# Find a release tarball
+def find_tarball
+  tarball = ENV['BEAKER_release_tarball']
+  tarball ||= Dir.glob('spec/fixtures/SIMP*.tar.gz')[0]
+  warn("Found Tarball: #{tarball}")
+  tarball
+end
+
+def tarball_yumrepos(tarball)
+  master.install_package('createrepo')
+  scp_to(master, tarball, '/root/')
+  tarball_basename = File.basename(tarball)
+  on(master, "mkdir -p /var/www && cd /var/www && tar xzf /root/#{tarball_basename}")
+  on(master, 'createrepo -q -p /var/www/SIMP/noarch')
+  create_remote_file(master, '/etc/yum.repos.d/simp_tarball.repo', <<-EOF.gsub(/^\s+/,'')
+    [simp-tarball]
+    name=Tarball repo
+    baseurl=file:///var/www/SIMP/noarch
+    enabled=1
+    gpgcheck=0
+    repo_gpgcheck=0
+    EOF
+  )
+  on(master, 'yum makecache')
+end
+
+describe 'install SIMP via rpm' do
+
+  masters = hosts_with_role(hosts, 'master')
+  agents  = hosts_with_role(hosts, 'agent')
+  let(:domain)      { fact_on(master, 'domain') }
+  let(:master_fqdn) { fact_on(master, 'fqdn') }
+
+
+  hosts.each do |host|
+    it 'should set the root password' do
+      on(host, "sed -i 's/enforce_for_root//g' /etc/pam.d/*")
+      on(host, 'echo password | passwd root --stdin')
+    end
+  end
+
+  context 'master' do
+    let(:simp_conf_template) { File.read(File.open('spec/acceptance/suites/rpm/files/simp_conf.yaml.erb')) }
+    masters.each do |master|
+      it 'should set up SIMP repositories' do
+        master.install_package('epel-release')
+        # master.install_package('http://yum.puppetlabs.com/puppetlabs-release-pc1-el-7.noarch.rpm')
+        # master.install_package('https://download.postgresql.org/pub/repos/yum/9.4/redhat/rhel-7-x86_64/pgdg-centos94-9.4-3.noarch.rpm')
+        on(master, 'curl -s https://packagecloud.io/install/repositories/simp-project/6_X_Dependencies/script.rpm.sh | bash')
+
+        tarball = find_tarball
+        if tarball.nil? or tarball.empty?
+          warn('='*72)
+          warn('Using Internet repos from packagecloud for testing')
+          warn('Specify a tarball with BEAKER_release_tarball or by placing one in spec/fixtures')
```

Comment:
If we're going to use the Internet anyway, we can pull tarballs from https://simp-project.com/ISO/SIMP/tar_bundles and we should probably support that method if possible.